sdcm/cluster.py

```diff
@@ -2359,6 +2359,15 @@ def prepare_files_for_archive(self, fileslist):
                 self.remoter.run('cp -r %s %s' % (f, new_full_path), ignore_status=True)
         return root_dir
 
+    def generate_coredump_file(self, restart_scylla=True):
+        self.log.info('Generate scylla core')
+        self.remoter.run("sudo pkill -f --signal 3 /usr/bin/scylla")
```

Comment:
thought, wouldn't it be helpful to coredump also the nodetool itself, and maybe scylla-jmx ? to have a wider picture of what is the issue ? (it won't always be in scylla side)

Reply:
the issue in drain is in Scylla... don't think it will be useful